contents/tutorials/vue-analytics.md

```diff
@@ -0,0 +1,281 @@
+---
+title: How to set up analytics in Vue with PostHog
```

Comment:
Should we include "with PostHog"  in the titie? We don't for others and I don't think it adds much (people will already see they are going to posthog.com).

Reply:
hmm yeah I see what you mean. I'll remove this in a different PR. I've done it for a bunch of them so will remove them all at once